src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,41 +1,55 @@
 // Copyright (c) FIRST and other WPILib contributors.
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
-
 package frc.robot;
-
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.XboxController;
 import frc.robot.commands.ExampleCommand;
 import frc.robot.subsystems.ExampleSubsystem;
+import frc.robot.subsystems.ShooterPrototypeSubsystem;
+import frc.robot.commands.RunShooterToRPMCommand;
 import edu.wpi.first.wpilibj2.command.Command;
-
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 /**
  * This class is where the bulk of the robot should be declared. Since Command-based is a
- * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
+ * “declarative” paradigm, very little robot logic should actually be handled in the {@link Robot}
  * periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
  * subsystems, commands, and button mappings) should be declared here.
  */
 public class RobotContainer {
-  // The robot's subsystems and commands are defined here...
+  // The robot’s subsystems and commands are defined here...
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
-
   private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
-
+  private final ShooterPrototypeSubsystem shooterTestSubsystem = new ShooterPrototypeSubsystem();
+  private XboxController gamepad = new XboxController(1);
+  private double shooterPower = 0.3;
```

Comment:
shooterPower should be owned by the subsystem. Maybe call it targetPower? Then create a getter so if other systems need it, they can call shooterSubsystem.getTargetPower() or something like that.